src/components/common/ProductBox/ProductBox.js

```diff
@@ -44,18 +44,31 @@ const ProductBox = ({ id, name, price, promo, stars }) => (
     <div className={styles.line}></div>
     <div className={styles.actions}>
       <div className={styles.outlines}>
-        <Button variant='outline'>
+        <Button compare={favorite} variant='outline'>
           <FontAwesomeIcon icon={faHeart}>Favorite</FontAwesomeIcon>
         </Button>
-        <Button variant='outline'>
+        <Button compare={compare} variant='outline'>
           <FontAwesomeIcon icon={faExchangeAlt}>Add to compare</FontAwesomeIcon>
         </Button>
       </div>
-      <div className={styles.price}>
-        <Button noHover variant='small'>
-          $ {price}
-        </Button>
-      </div>
+
+      {oldPrice ? (
+        <div className={styles.price}>
+          <Button noHover variant='oldPrice'>
+            $ {oldPrice}
+          </Button>
+
+          <Button noHover variant='small'>
```

Comment:
Hej! Jeśli zwykły price ma być zawsze, możesz zawsze zrobić warunek:

`{
oldPrice &&
<Button dla oldPrice>
}`

Jeśli oldPrice będzie, to stara cena się pokaże, jeśli nie, to zniknie :)